plugins/polio/api.py

```diff
@@ -513,6 +517,15 @@ def send_notification_email(self, request, pk, **kwargs):
 
         return Response({"message": "email sent"})
 
+    # We need to authorize PATCH request to enable restore_deleted_campaign endpoint
+    # But Patching the campign directly is very much error prone, so we disable it indirectly
+    def partial_update(self):
```

Comment:
This works, and is ok for this, but I'm quite sure that there is a cleaner way to disallow a method on a viewset: https://stackoverflow.com/questions/9549867/django-rest-framework-limit-the-allowed-methods-to-get